cmd/csi-addons/main.go

```diff
@@ -56,6 +59,9 @@ func init() {
 	flag.StringVar(&cmd.operation, "operation", "", "csi-addons operation")
 	flag.StringVar(&cmd.persistentVolume, "persistentvolume", "", "name of the PersistentVolume")
 	flag.StringVar(&cmd.drivername, "drivername", "", "name of the CSI driver")
+	flag.StringVar(&cmd.secret, "secret", "", "kubernetes secret in the format `namespace/name`")
+	flag.StringVar(&cmd.cidrs, "cidrs", "", "comma separated list of cidrs to fence/unfence")
```

Comment:
no need to mention fence/unfence, possibly these parameters will be used for other operations in the future

Reply:
Done.